plugins/ProfileExtender/views/profilefields.php

```diff
@@ -3,8 +3,19 @@
 if (is_array($this->ProfileFields)) {
     foreach ($this->ProfileFields as $Name => $Field) {
         $Options = array();
-        if ($Field['FormType'] == 'Dropdown')
-            $Options = array_combine($Field['Options'], $Field['Options']);
+        if ($Field['FormType'] == 'Dropdown') {
+            $values = $Field['Options'];
+            $labels = (array_key_exists('OptionsLabels', $Field)) ? $Field['OptionsLabels'] : false;
```

Comment:
Simplify all this:
     $Values = $Field['Options'];
     $Labels = val('OptionsLabels', $Field, $Field['Options']);
     $Options = array_combine($Values, $Labels);